feat: Shared `SKPicture` recording

MartinZikmund opened the pull request:
**GitHub Issue:** closes https://github.com/unoplatform/uno-private/issues/1427, closes https://github.com/unoplatform/uno/issues/21130

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal), unless the change is documentation related. -->

## PR Type: 🐞 Bugfix

<!--
Copy the labels that apply to this PR and paste them above:

- 
- ✨ Feature
- 🎨 Code style update (formatting)
- 🔄 Refactoring (no functional changes, no api changes)
- 🏗️ Build or CI related changes
- 📚 Documentation content changes
- 🤖 Project automation
- 💬 Other... (Please describe)

-->


## What is the current behavior? 🤔

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->


## What is the new behavior? 🚀

<!-- Please describe the new behavior after your modifications. -->

## PR Checklist ✅

Please check if your PR fulfills the following requirements:

- [x] 📝 Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
- [ ] 🧪 Added [Runtime tests, UI tests, or a manual test sample](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] 📚 Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] 🖼️ Validated PR `Screenshots Compare Test Run` results.
- [x] ❗ Contains **NO** breaking changes

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information ℹ️

<!-- Please provide any additional information if necessary -->

MartinZikmund committed:
feat: Shared `SKPicture` recording

MartinZikmund committed:
refactor: Use shared `SKPicture` recording on X11

MartinZikmund committed:
feat: Use shared `SKPicture` recording on WASM

MartinZikmund committed:
feat: Use shared `SKPicture` recording on macOS

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Adjust to run correctly on macOS

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

spouliot commented on src/Uno.UI.Runtime.Skia.MacOS/UI/Xaml/Window/MacOSWindowHost.cs:
```diff
@@ -78,40 +78,40 @@ public MacOSWindowHost(MacOSWindowNative nativeWindow, Window winUIWindow, XamlR
+			canvas.Clear(SKColors.Transparent);
 
-			if (RootElement?.Visual is { } rootVisual)
+			int width = (int)nativeWidth;
```
static analysis warning is legit

spouliot commented on src/Uno.UI.Runtime.Skia.MacOS/UI/Xaml/Window/MacOSWindowHost.cs:
```diff
@@ -78,40 +78,40 @@ public MacOSWindowHost(MacOSWindowNative nativeWindow, Window winUIWindow, XamlR
 
-			if (RootElement?.Visual is { } rootVisual)
+			int width = (int)nativeWidth;
+			int height = (int)nativeHeight;
```
static analysis warning is legit

MartinZikmund committed:
refactor: Shared picture rendering

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Adjust rendering on WASM

MartinZikmund committed:
feat: Split rendering on iOS

MartinZikmund committed:
refactor: Adjustments to split rendering API

MartinZikmund committed:
refactor: Shared split rendering on X11

MartinZikmund committed:
chore: Further API changes

MartinZikmund committed:
refactor: Split rendering on WPF

MartinZikmund committed:
refactor: Split rendering on Win32

MartinZikmund committed:
refactor: Cleanup

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

MartinZikmund committed:
chore: Win32 adjustments

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

MartinZikmund committed:
chore: Adjust for Islands

MartinZikmund committed:
chore: Adjustments for Android

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Adjust iOS

MartinZikmund committed:
chore: iOS validation

MartinZikmund committed:
chore: Adjust WASM

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
fix: Ensure scale is updated along with window size

Previously we updated scale only after window size change, which meant that in some cases it did not invalidate render. If the scale changed, we now apply it right before size change, which means it will be up to date.

MartinZikmund committed:
feat: Set tree render dirty on rasterization scale change

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Move path rescaling

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Improve resizing accuracy

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

MartinZikmund committed:
chore: Move method

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
⚠️⚠️ The build [173019](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=173019) has failed on Uno.UI - CI.

MartinZikmund committed:
chore: Formatting

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

ramezgerges committed:
chore: fix invalidation bug

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Undo positioning change

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
⚠️⚠️ The build [173066](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=173066) has failed on Uno.UI - CI.

MartinZikmund committed:
chore: Cleanup

MartinZikmund committed:
fix: Correctly enqueue the invalidation

MartinZikmund committed:
chore: Idle priority

MartinZikmund committed:
chore: Adjust priority for Islands

MartinZikmund committed:
fix: Use `QueueInvalidateRender` everywhere

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21189/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

unodevops commented:
⚠️⚠️ The build [173239](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=173239) has failed on Uno.UI - CI.

MartinZikmund committed:
chore: Adjust test check for iOS

MartinZikmund committed:
chore: Address comment

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

MartinZikmund committed:
chore: Adjust tvOS

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21189/docs/index.html

nventive-devops commented:
The [build 173268](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=173268) found UI Test snapshots differences: `android-28-net8`: **30**, `android-28-net8-Snap`: **44**, `ios`: **4**, `ios-Snap`: **48**, `skia-linux-screenshots`: **86**, `skia-windows-screenshots`: **177**, `wasm`: **58**, `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **1**, `wasm-automated-net9.0-WinUI-Default-automated`: **16**, `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **1**

<details>
<summary>Details</summary>


* `android-28-net8`: **30** changed over 825

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `DecimalFormatterTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Detereminate_ProgressRing_Validation50_[#FF0000_#008000_#008000_#FF0000]_Progress-Ring-Value-50`
  - `Detereminate_ProgressRing_Validation0_[#FF0000_#FF0000_#FF0000_#FF0000]_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIDeterminateProgressRing`
  - `Detereminate_ProgressRing_Validation25_[#FF0000_#008000_#FF0000_#FF0000]_Progress-Ring-Value-25`
  - `DoesNotInterfereWithHorizontalDrag_UITests_Microsoft_UI_Xaml_Controls_RefreshContainerTests_RefreshContainerHorizontalScroll`
  - `FlyoutTest_When_NoOverlayInputPassThroughElement_Then_DontPassThrough_woOff_UITests_Shared_Windows_UI_Xaml_Controls_Flyout_Flyout_OverlayInputPassThroughElement`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `NavigateBackAndForthBetweenMenuItemsAndSettings_Fluent_SamplesApp_Samples_Microsoft_UI_Xaml_Controls_NavigationViewTests_FluentStyle_FluentStyle_NavigationViewSample`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `SelectionOnLoad_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsInitialLoadSelected`
  - `SelectionTest_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage`
  - `WebView_NavigateToAnchor_click_anchor`
  - `When_Clip_Is_Set_On_Child_Element_original`
  - `WebView_NavigateToAnchor_Initial`
  - `When_Clip_Is_Set_On_Child_Element_validation`
  - `WebView_NavigateToAnchor_navigate_to_anchor`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `UpDownTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `android-28-net8-Snap`: **44** changed over 1074

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Gesture_Recognizer_Pointer_Events_test_bench_UITests_Shared_Windows_UI_Input_GestureRecognizer_PointersEvents`
  - `MUX_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage`
  - `Scrolling_MUXControlsTestApp_ScrollViewDynamicPage_MUXControlsTestApp_ScrollViewDynamicPage`
  - `Scrolling_MUXControlsTestApp_ScrollViewKeyboardAndGamepadNavigationPage_MUXControlsTestApp_ScrollViewKeyboardAndGamepadNavigationPage`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `TextBox_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_Width_Affects_Delete_Button_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_Width_Affects_Delete_Button`
  - `TextBox_UITests_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBox_AutoFill_UITests_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBox_AutoFill`
  - `TextBox_UITests_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBox_iOS_DuplicatingText_UITests_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBox_iOS_DuplicatingText`
  - `Border_Border_AntiAlias_UITests_Windows_UI_Xaml_Controls_BorderTests_BorderAntiAlias`
  - `MUX_NumberBox_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `TextBox_UITests_Shared_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBoxPage_UITests_Shared_Windows_UI_Xaml_Controls_PasswordBoxTests_PasswordBoxPage`
  - `TextBox_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxControl_TextBox_Binding_Null_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxControl_TextBox_Binding_Null`
  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `MediaPlayerElement_Using_3gp_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_3gp_Extension`
  - `MediaPlayerElement_Using_mp3_Audio_only_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mp3_Extension`
  - `MediaPlayerElement_Using_ogg_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Ogg_Extension`
  - `Border_Border_Corner_Overlap_UITests_Shared_Windows_UI_Xaml_Controls_BorderTests_Border_CornerOverlap`
  - `Microsoft_UI_Composition_UITests_Shared_Windows_UI_Composition_SKCanvasElement_Simple_UITests_Shared_Windows_UI_Composition_SKCanvasElement_Simple`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `Transform_Basics_UITests_Shared_Windows_UI_Xaml_Media_Transform_Basics`
  </details>

* `ios`: **4** changed over 256

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `TextBox_UpdatedBinding_On_OneWay_Mode_UITests_Windows_UI_Xaml_Controls_TextBox_TextBox_Bindings`
  - `Check_ListView_Swallows_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  </details>

* `ios-Snap`: **48** changed over 991

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ListView_ListView_Aligned_Left_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Aligned_Left`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `Flyouts_UITests_Windows_UI_Xaml_Controls_Flyout_Flyout_TemplatedParent_UITests_Windows_UI_Xaml_Controls_Flyout_Flyout_TemplatedParent`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Alignment_SizeOnControl_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Alignment_SizeOnControl`
  - `NavigationView_MUXControlsTestApp_NavigationViewMenuItemStretchPage_MUXControlsTestApp_NavigationViewMenuItemStretchPage`
  - `NavigationView_MUXControlsTestApp_NavigationViewMinimalPage_MUXControlsTestApp_NavigationViewMinimalPage`
  - `RatingControl_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlBasic_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlBasic`
  - `RatingControl_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage`
  - `Brushes_BorderImageBrush_Uno_UI_Samples_UITests_ImageBrushTestControl_BorderImageBrush`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_MixTransformCliPropertyAndClippedByParentWithBorders_Then_RenderingIsValid_UITests_Windows_UI_Composition_MixTransformCliPropertyAndClippedByParentWithBorders_Then_RenderingIsValid`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_TransformElementClippedByParentWithBorder_Then_ClippingAppliedPostRendering_UITests_Windows_UI_Composition_TransformElementClippedByParentWithBorder_Then_ClippingAppliedPostRendering`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_TransformElementClippedByParent_Then_ClippingAppliedPostRendering_UITests_Windows_UI_Composition_TransformElementClippedByParent_Then_ClippingAppliedPostRendering`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_Vector3KeyFrameAnimationSample_UITests_Windows_UI_Composition_Vector3KeyFrameAnimationSample`
  - `SwipeControl_MUXControlsTestApp_SwipeControlPage2_MUXControlsTestApp_SwipeControlPage2`
  - `GridView_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped`
  - `NavigationView_NavigationViewSample_SamplesApp_Samples_NavigationViewSample_NavigationViewSample`
  - `ContentControl_ContentControl_SelectorInheritance_Uno_UI_Samples_Content_UITests_ContentControlTestsControl_ContentControl_SelectorInheritance`
  - `Grid_CenteredGridinGridwiththreefixedsizechildren_Uno_UI_Samples_Content_UITests_GridTestsControl_CenteredGridinGridwiththreefixedsizechildren`
  </details>

* `skia-linux-screenshots`: **86** changed over 2282

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Buttons.png`
  - `CalendarView_Theming.png-dark`
  - `ExpanderColorValidationPage.png-dark`
  - `ExpanderColorValidationPage.png`
  - `Focus_FocusVisual_Properties.png-dark`
  - `Focus_FocusVisual_Properties.png`
  - `Examples.png-dark`
  - `Examples.png`
  - `ButtonClippingTestsControl.png`
  - `Buttons.png-dark`
  - `ImageIconPage.png-dark`
  - `ImageIconPage.png`
  - `Gamepad_CurrentReading.png-dark`
  - `Gamepad_CurrentReading.png`
  - `DisplayInformation.png-dark`
  - `DisplayInformation.png`
  - `DoubleImageBrushInList.png-dark`
  - `DoubleImageBrushInList.png`
  - `Gamepad_Enumeration.png-dark`
  - `Gamepad_Enumeration.png`
  </details>

* `skia-windows-screenshots`: **177** changed over 2282

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ButtonClippingTestsControl.png-dark`
  - `ButtonClippingTestsControl.png`
  - `Buttons.png-dark`
  - `Border_With_Off_Centre_RotateTransform.png`
  - `Border_With_Off_Centre_ScaleTransform.png-dark`
  - `Border_With_ScaleTransform.png`
  - `Border_With_TranslateTransform.png-dark`
  - `CompositionEffectBrush.png-dark`
  - `CompositionEffectBrush.png`
  - `Basics_Automated.png-dark`
  - `Basics_Automated.png`
  - `Buttons.png`
  - `ClipboardTests.png-dark`
  - `ClipboardTests.png`
  - `ImageBrush_SameWithDelay.png-dark`
  - `ImageBrush_SameWithDelay.png`
  - `ImageBrush_Stretch_None.png-dark`
  - `ImageBrush_Stretch_None.png`
  - `ImageBrushInList.png-dark`
  - `ImageBrushInList.png`
  </details>

* `wasm`: **58** changed over 1056

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_DataTemplateSelector`
  - `Uno.UI.Samples.Content.UITests.GridView.GridViewVariableItemHeight`
  - `GenericApp.Views.Content.UITests.ContentControlTestsControl.ContentControl_UnsetContent`
  - `MUXControlsTestApp.SwipeControlPage2`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_WithScrollViewer`
  - `UITests.Microsoft_UI_Xaml_Controls.WebView2Tests.WebView2_NavigationProperties`
  - `UITests.Shared.Windows_UI_Xaml_Controls.CommandBar.CommandBar_Xaml_Automated`
  - `Uno.UI.Samples.Content.UITests.ContentControlTestsControl.ContentControl_WithPadding`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.UndefinedHeightListView`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ComboBox.ComboBox_VisibleBounds`
  - `UITests.Windows_UI_Xaml_Controls.SwipeControlTests.SwipeControl_ListView_ItemClick`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Minimal`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Ogg_Extension`
  - `UITests.Uno_Web.Http.CookieManagerTests`
  - `initial_state`
  - `UITests.Microsoft_UI_Xaml_Controls.TabViewTests.TabViewPage`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_Weird_Measure`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_With_ListViews_Count_Measure`
  - `UITests.Windows_UI_Xaml_Controls.ComboBox.ComboBox_Stretch`
  - `MUXControlsTestApp.TextCommandBarFlyoutPage`
  </details>

* `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunBenchmarks_Benchmarks_Shared_Controls_BenchmarkDotNetTestsPage`
  </details>

* `wasm-automated-net9.0-WinUI-Default-automated`: **16** changed over 877

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Default_StrokeThickness_MyRect`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `Default_StrokeThickness_MyLine`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `When_NoSelection_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Default_StrokeThickness_MyPath`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ToggleCollapsedStateToVerifyClipping_UITests_Shared_Microsoft_UI_Xaml_Controls_ExpanderTests_ExpanderColorValidationPage`
  - `When_NoSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_Theme_Changed_No_Crash_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Default_StrokeThickness_MyEllipse`
  - `Default_StrokeThickness_MyPolyline`
  - `Default_StrokeThickness_MyPolygon`
  - `TestProgressRing_InitialState_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunRuntimeTests_SamplesApp_Samples_UnitTests_UnitTestsPage`
  </details>

</details>
